pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -53,6 +53,37 @@ func (cm *containerManagerImpl) createNodeAllocatableCgroups() error {
 	return nil
 }
 
+func (cm *containerManagerImpl) updateInternalCapacity(rl v1.ResourceList) bool {
```

Comment:
This func doesn't do the update.
Maybe rename it shouldUpdateInternalCapacity

Reply:
Technically the function updates `cm.internalCapacity`, hence the name.